src/webextension/survey/components/app.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Uri from 'urijs';
+
+import Survey from './survey.jsx';
+
+class App extends React.Component {
+  // Read intiial form values from querystring, ensuring that the `sentiment`
+  // value is a number.
+  getInitialValues() {
+    const values = Object.assign({
+      details: ''
+    }, new Uri(window.location.search).search(true));
+    if ('sentiment' in values) {
+      values.sentiment = parseInt(values.sentiment);
```

Comment:
nit: you should probably pass a radix to `parseInt()` due to occasional unexpected behavior (although not like we're expecting leading zero input): http://eslint.org/docs/rules/radix